docs-raw/site/docs/fabric-gateway-features.md

```diff
@@ -1,24 +1,24 @@
 # Fabric Gateway
 
-Powered by [Skipper](https://github.com/zalando/skipper), Fabric Gateway is a security-audited and scalable solution that makes it easy for developers to secure and monitor APIs at scale.
+Fabric Gateway is a Kubernetes control plane for [Skipper](https://github.com/zalando/skipper), the open-source HTTP router. It delivers a feature-rich API gateway that secures applications at the point of ingress to a Kubernetes cluster. It is a security-audited and scalable solution that makes it easy for developers to secure and monitor APIs at scale.
 
 ![Gateway only](img/gateway-only.png)
 
-The Gateway operates at the Kubernetes Ingress layer and is capable of providing the above features (and more) for all inbound traffic, so application developers don't have to.
-A companion [Grafana dashboard](https://zmon.zalando.net/grafana/dashboard/db/fabric) allows developers to monitor their Gateway-secured APIs with fine-grained per-client performance visibility.
+## Configuration
 
-To enable the Fabric Gateway for an application simply add this Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) to its CDP setup (e.g., by placing in `deploy/apply`). **N.B.** You should not have both a `FabricGateway` and a standard `Ingress` resource defined, as the default ingress would not give you any authentication but would allow traffic access to your unprotected backend service. If you are using `FabricGateway` to manage auth for your service, then please remove any existing ingress resources.
+Fabric Gateway is configured via a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) of kind `FabricGateway`. To enable for an application create the `FabricGateway` resource in your cluster. If using CDP, this can be achieved by placing the `FabricGateway` resource into your `deploy/apply` folder. 
```

Comment:
Seeing as these are public docs as well, we prob shouldnt be referencing CDP... Or anything internal.